pgxn/neon/communicator.c

```diff
@@ -1098,7 +1098,7 @@ prefetch_register_bufferv(BufferTag tag, neon_request_lsns *frlsns,
 						  BlockNumber nblocks, const bits8 *mask,
 						  bool is_prefetch)
 {
-	uint64		min_ring_index;
+	uint64		ring_index;
```

Comment:
As for PR's description, maybe we should rename it to last_slot_index?

Reply:
Changed